company.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script type="module" src="script/company.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>    
+        <script type="module">
+            import {CompanyChart} from '../script/company.js'
+            new CompanyChart
+            
+        </script> 
```

Comment:
This should be imported at the bottom of the page.
Also, this should be a in a different js file and import the file instead